### PR TITLE
First Capstone Project 

### PR DESCRIPTION
**Pull request by abeltsew**
Dear Reviewer,

In this PR the complete project implementation is included. The Main features completed are listed below.

- [ ] Home Page
- [ ] About Page
- [ ] Making the web pages responsive
- [ ] Loading in the Featured Speakers section Dynamically using JS
- [ ] Link navigation between the Home and about page
- [ ] mobile Menu popup design and implementation

Please find the Video presentation link [Here](https://www.loom.com/share/d77960348a0a4261b4e17103ae82540e)

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -0,0 +1,182 @@
...
+          </p>
+        </li>
+      </ul>
+      <button class="get-ticket">Get your ticket Here</button>
+    </section>
+    <!-- Guest section -->
+    <section class="guests" id="guests">
+      <h2>Featured Actors</h2>
+      <div class="guests-list"></div>
+    </section>
```
Comment (muneebulrehman) [inline index.html:155]: Great job with the design. However, this section doesn't quite match the Behance design. I would recommend scaling down your image in height, so as to fit the text also.

| Your design  | Behance design |
| ------------- | ------------- |
| image | image |

File: index.html [hunk 2]
```diff
@@ -0,0 +1,182 @@
...
+      <ul class="partners">
+        <li>
+          <img src="assets/images/warner-bros.jpeg" alt="Warner Brothers" />
+        </li>
+        <li><img src="assets//images/hbo.png" alt="Hbo max" /></li>
+        <li><img src="assets/images/netflix.png" alt="Netflix" /></li>
+        <li><img src="assets/images/apple-tv.png" alt="Apple" /></li>
+        <li><img src="assets/images/roku.png" alt="Roke" /></li>
+      </ul>
+    </section>
```
Comment (muneebulrehman) [inline index.html]: I would recommend having the same background color in this section as in the Behance design. Kindly do so.

| Your design  | Behance design |
| ------------- | ------------- |
| image| image |

**Review by muneebulrehman — CHANGES_REQUESTED** [review]
Hi @abeltsew , 

Good job so far! I see you put a lot of effort into this project. However, there are some issues that you still need to work on to go to the next project but you are almost there!

## Highlights
- No linter errors
- Descriptive PR
- Good presentation
- Great design and theme for the project.

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by Olamarx — APPROVED** [review]
Hi @abeltsew,

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

## Highlights
- Linters passed ✔️ 
- Behance resemblance ✔️
- Descriptive PR ✔️

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**